charts/kubernetes-agent/migrations.md

```diff
@@ -0,0 +1,38 @@
+# Migrations
+## Version 1 --> 2
```

Comment:
A small preamble before this heading about what the migrations are and why they are needed. Just to give some context to the rest of this doc